src/Module/Events/Export.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2022, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Module\Events;
```

Comment:
I am not happy that the namespace is plural when only one other is, do you mind replacing with `Calendar`, including the `Events\Json` Module class?

Reply:
Hehe, I was at the same point yesterday, but I thought it's maybe intended that it's called events and not calendar.

Reply:
I don't think so. There were two files in the `mod/` folder called `cal.php` and `events.php` but we aren't limited by the flat structure anymore.